test/e2e/storage/volumes.go

```diff
@@ -260,7 +260,7 @@ var _ = SIGDescribe("Volumes", func() {
 				Namespace:   namespace.Name,
 				Prefix:      "cephfs",
 				ServerImage: framework.CephServerImage,
-				ServerPorts: []int{6789},
+				ServerPorts: []int{6789, 6808},
```

Comment:
6808 is probably accidental, the osd port ranges 6800:7300
http://docs.ceph.com/docs/hammer/rados/configuration/network-config-ref/

Reply:
Fixed